### DISCUSSION TITLE
Is it supposed to show seams in the uv map when using IBL?

### DISCUSSION THREAD

**Discussion by jnmaloney** [google/filament#3365] [General]
I tried unwrapping a model using *lightmap pack* in Blender, and then the lighting was all messed up in Filament. So I tried a different uv unwrap and used Filament material with IBL only. The lighting shows the uv seam on the model.

Is this expected and if so how can i fix my models?

![Screenshot from 2020-12-15 10-11-47](https://user-images.githubusercontent.com/5510983/102151763-4b6c5a80-3ebf-11eb-8bb7-082eedcee330.png)

Edit: The problem can go away if I compile new materials instead of using textured.filamat.

**Comment by bejado** [comment]
Can you link the Blender file you're using?

**Comment by jnmaloney** [comment]
Ok [simple-heart-modifiied.zip](https://github.com/google/filament/files/5705802/simple-heart-modifiied.zip)

I'm still getting used to Filament, but I think that it is not supposed to look like that and I made some mistake with the material, because I was using the default textured material without giving a normal map.
